### PR TITLE
WIP DO NOT MERGE: feat: new booking flow initated

### PR DESCRIPTION
**Pull request by ckeyes88**
https://github.com/HelpfulHuman/ShopifyExperiencesApp/issues/6371

### CODE REVIEW

File: src/fullPageBookingForm/Components/App.tsx [hunk 1]
```diff
@@ -51,7 +63,11 @@ export type AppProps = {
   secondaryColor?: string;
   textColor?: string;
   buttonShape?: string;
-  disabledColor?: string
+  disabledColor?: string;
+  eventUnit: string;
```
Comment (ckeyes88) [inline src/fullPageBookingForm/Components/App.tsx:67]: @farhan711 this should come from custom labels not be passed in. It's in custom labels as per person or per unit label.

**Review by ckeyes88 — COMMENTED** [review]
@farhan711 I did this PR to see the diff and make comments. I think this is moving in the right direction. I think what would be most helpful for @NickGlenn and @markgustav is if you provided some image and/or video demos to display the progress. That way they can redirect if needed and make sure you're understanding the main outcome.

**Comment by markgustav** [comment]
That's an interesting approach @farhan711. Can we make that an option in the section settings in the Shopify 2.0 theme editor? 
option 1: No outline 
option 2: Radiused outline - color/thickness settings for the outline
option 3: Square outline - color/thickness settings for the outline

**Comment by markgustav** [comment]
Ok. That's cool @farhan711. When we're ready to roll out the new onboarding experience for new Experiences customers, how close are you to having the new booking button ready for that? @NickGlenn and @ckeyes88 - any thoughts/questions for Farhan are welcome.

File: src/fullPageBookingForm/Components/App.tsx (hunk 1 above)
Comment (ckeyes88) [reply src/fullPageBookingForm/Components/App.tsx:67]: @farhan711 the labels come from the "customLabels" endpoint in `src/Shared/Utils/api.ts`

```ts
/**
 * Fetches custom event labels set in admin interface of an experience
 */
export async function getEventCustomLabels({
  baseUrl,
  shopId,
  shopifyProductId,
}: GetEventArgs): Promise<GetEventCustomLabelsResponse> {
  const res = await axios.get<
    GetCustomFormRequestBody,
    AxiosResponse<GetEventCustomLabelsResponse>
  >(
    `${baseUrl}/rest/event/custom-labels?productId=${shopifyProductId}&shop=${shopId}`,
  );

  return res.data;
}
```

**Comment by markgustav** [comment]
@farhan711 Are you waiting on someone to QA or merge? @NickGlenn, have you seen this?

**Comment by NickGlenn** [comment]
@markgustav I've seen this, but its marked as WIP DO NOT MERGE - is it ready to be merged?

**Comment by markgustav** [comment]
@NickGlenn No, I don't think so. @farhan711 can you give a report on this? How can we move it forward?

**Comment by farhan711** [comment]
I think its done, let me know if there's any changes required as per above.

**Comment by markgustav** [comment]
Ok. @NickGlenn this is on us to review now.

**Comment by NickGlenn** [comment]
@markgustav Will do after my dentist appointment

**Review by NickGlenn — APPROVED** [review]
I don't see any obvious issues, looks good.

**Pull request merged**